tests/functional/splinter_test.c

```diff
@@ -977,8 +1016,9 @@ splinter_perf_inserts(platform_heap_id             hid,
    timestamp insert_latency_max = 0;
    uint64    read_io_bytes, write_io_bytes;
    cache_io_stats(cc[0], &read_io_bytes, &write_io_bytes);
-   uint64 io_mib    = (read_io_bytes + write_io_bytes) / MiB;
-   uint64 bandwidth = io_mib / NSEC_TO_SEC(total_time);
+   uint64 io_mib = (read_io_bytes + write_io_bytes) / MiB;
+   uint64 bandwidth =
+      (NSEC_TO_SEC(total_time) ? (io_mib / NSEC_TO_SEC(total_time)) : 0);
```

Comment:
Changes like this one fix divide-by-zero errors that occur if you run with very small `--num-inserts 10000` arg.

Now, the entire test can run with small # of rows, which greatly helps with re-validating test-code changes.